### PR TITLE
[IMPROVEMENT] Add Model for blacklisted users

### PR DESCRIPTION
**Pull request by MatejMecka**
Please prefix your pull request with one of the following: **[FEATURE]** **[FIX]** **[IMPROVEMENT]**.

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/CCExtractor/sample-platform/blob/master/.github/CONTRIBUTING.md).
- [ ] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [x] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

---
I created a model for blacklisted users. The model has the properties 
* user_id the user id github has each for a unique user. 
* username which is the username of that user which is also unique  
* name which in most cases is the first last name of the user. 

Those 2 serve for recognizing which user has been blocked much more easily

### CODE REVIEW

File: mod_ci/models.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 """
 
 import datetime
-from sqlalchemy import Column, Integer, String, ForeignKey, DateTime, Boolean
+from sqlalchemy import Column, Integer, String, ForeignKey, DateTime, Booleana
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: Why this change?

File: mod_ci/models.py [hunk 2]
```diff
@@ -83,3 +83,17 @@ def __repr__(self):
...
         """
         return '<Platform {p}, maintenance {status}>'.format(p=self.platform.description, status=self.disabled)
+
+
+class blackisted_users(Base):
+    __tablename__ = 'BlockedUsers'
+    # user_id is the user's id GitHub assigns.
+    user_id = Column(Integer, primary_key=True)
+    # username refers to the username of the user.
+    username = Column(String(41), unique=True)
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: Why do we need this?

File: mod_ci/models.py [hunk 3]
```diff
@@ -83,3 +83,17 @@ def __repr__(self):
...
+    # name refers to the name property from GitHub's api.
+    name = Column(String(64))
+
+    def __repr__(self):
+        return "<blacklisted_users(user_id='{}',name='{}',username="{}")>".format(
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: Use named arguments please.

File: mod_ci/models.py (hunk 2 above)
Comment (MatejMecka) [reply mod_ci/models.py]: Because if we only have user id it'll be confusing. The addition of Username and Name serve for better categorizing and finding users easier

Comment (canihavesomecoffee) [reply mod_ci/models.py]: But those can be retrieved by using the GitHub API... If you store them and a user changes name, the information is outdated.

Comment (MatejMecka) [reply mod_ci/models.py]: If you want I can only leave username and delete name?

File: mod_ci/models.py [hunk 4]
```diff
@@ -94,6 +94,6 @@ class blackisted_users(Base):
     # name refers to the name property from GitHub's api.
     name = Column(String(64))
 
-    def __repr__(self):
+    def __repr__(self,name,username,user_id):
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: That's not "named arguments". I meant named arguments for the format.

File: mod_ci/models.py (hunk 2 above)
Comment (MatejMecka) [reply mod_ci/models.py]: The userid will not change but if it's the same id and the username or name is different change the record

File: mod_ci/models.py [hunk 5]
```diff
@@ -7,6 +7,16 @@
  'Maintenance mode' => 'maintenance_mode']
 """
 
+import datetime
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: What's this doing here?

File: mod_ci/models.py [hunk 6]
```diff
@@ -19,7 +29,8 @@ class Kvm(Base):
     __tablename__ = 'kvm'
     __table_args__ = {'mysql_engine': 'InnoDB'}
     name = Column(String(64), primary_key=True)
-    test_id = Column(Integer, ForeignKey(Test.id, onupdate="CASCADE", ondelete="RESTRICT"))
+    test_id = Column(Integer, ForeignKey(
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: Why this change?

File: mod_ci/models.py [hunk 7]
```diff
@@ -83,3 +94,18 @@ def __repr__(self):
         :rtype str(platform, status): str
         """
         return '<Platform {p}, maintenance {status}>'.format(p=self.platform.description, status=self.disabled)
+
+
+class blacklisted_users(Base):
```
Comment (canihavesomecoffee) [inline mod_ci/models.py]: Not CamelCase class name.

**Comment by canihavesomecoffee** [comment]
Closing this as I merged the other one. Good job though 👍
